finesse/hardware/__init__.py

```diff
@@ -4,14 +4,17 @@
 from pubsub import pub
 
 if "--dummy-em27" in sys.argv:
+    from .dummy_em27_scraper import DummyEM27Scraper as EM27Scraper
     from .opus.dummy import DummyOPUSInterface as OPUSInterface
 else:
+    from .em27_scraper import EM27Scraper  # type: ignore
     from .opus.em27 import OPUSInterface  # type: ignore
 
 from .stepper_motor import create_stepper_motor_serial_manager
 from .temperature import create_temperature_controller_serial_managers
 
 opus: OPUSInterface
+scraper: EM27Scraper
```

Comment:
As you're defining `scraper` below, you do not need to assign a type to it here.